test.js

```diff
@@ -50,3 +50,7 @@ test('supports non-english characters', t => {
 
 	</svg>`));
 });
+
+test('support markup inside Entity tags', t => {
+	t.true(isSvg('<?xml version="1.0" encoding="utf-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd" [ <!ENTITY ns_flows "http://ns.adobe.com/Flows/1.0/"> <!ENTITY Smile " <rect x=\'.5\' y=\'.5\' width=\'29\' height=\'39\' fill=\'black\' stroke=\'red\'/> <g transform=\'translate(0, 5)\'> <circle cx=\'15\' cy=\'15\' r=\'10\' fill=\'yellow\'/><circle cx=\'12\' cy=\'12\' r=\'1.5\' fill=\'black\'/><circle cx=\'17\' cy=\'12\' r=\'1.5\' fill=\'black\'/><path d=\'M 10 19 L 15 23 20 19\' stroke=\'black\' stroke-width=\'2\'/></g>"> ]><svg></svg>'));
```

Comment:
I think it could be useful with some more assertions to ensure the entity regex covers all the ways Entity could be used and what it could contain.

Reply:
There were already some tests covering regular entity tags on validity tests. 
I added some regarding markup only:
- svg with single entity tag with markup;
- svg with multiple entity tags (no-markup and with markup)
- svg with multiple entity tags both with markup)